Component versions (#86)
mulesoft/docs-private-cloud: modules/ROOT/pages/config-add-proxy-whitelist.adoc

To add whitelist addresses:

. Log in to Ops Center and select *Configuration*.
. Select the `api-manager` namespace from the right drop-down menu and then select the `api-platform-web-env` config map from the left drop-down menu.

> . Select *APW_FEATURES_PROXYWHITELIST* and ensure that value is set to `true`. If not, set it to `true` and then select `Apply`.

Reviewer: Please follow the style reference and my email reminders and limit list items to one sentence max, as follows:
```suggestion
. Select *APW_FEATURES_PROXYWHITELIST* and ensure that value is set to `true`. 
+
+ If not, set it to `true` and then select `Apply`.
```
OR
```suggestion
. Select *APW_FEATURES_PROXYWHITELIST* and, if it is not already, set its value to `true` and then select `Apply`.
```

. Select *APW_PROXYWHITELIST_RULES*.

> . Write your rules separated by commas. The following example defines regular expressions that allow requests to be made to the `*.somewhere.com/*` and `*.somewhereelse.com/*` domains, where `*` is any part of a DNS name or URL:

Reviewer: ditto and GLOBAL re. list item limits:
```suggestion
. Write your rules, separated by commas. 
+
+ The following example defines regular expressions that allow requests to be made to the `*.somewhere.com/*` and `*.somewhereelse.com/*` domains, where `*` is any part of a DNS name or URL:
```

----
.*\.somewhere\.com,.*\.somewhereelse\.com 
----

. Select *Apply* to save changes to the `api-platform-web-env` config map.

> . Recreate the pods to ensure each node in the cluster uses the most recent configuration:

Reviewer: The one major exception to not hyphenating a prefix, because the meaning is incorrect with this presentation:
```suggestion
. Re-create the pods to ensure that each node in the cluster uses the most recent configuration:
```
